Application/EdFi.Ods.Admin/Initialization/NamespacePrefixInitializationModel.cs

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+namespace EdFi.Ods.Admin.Initialization
+{
+    public class NamespacePrefixInitializationModel
+    {
+        public string Name { get; set; }
```

Comment:
This should probably be named "NamespacePrefix" to be clear what it represents, we're not naming the prefixes.

Reply:
done